cypress/e2e/shared/arduino.test.ts

```diff
@@ -0,0 +1,83 @@
+describe('Arduino onboarding', () => {
+  beforeEach(() => {
+    cy.flush()
+    cy.signin()
+    cy.setFeatureFlags({firstMile: true, onboardArduino: true})
+  })
+
+  it('should load the main home-page that has the arduino tile', function() {
+    cy.getByTestID('home-page--header').should('exist')
+  })
+
+  describe('Arduino', () => {
+    beforeEach(() => {
+      cy.getByTestID('homepage-wizard-language-tile--arduino').click()
+    })
```

Comment:
since there's only one `it` block in this `describe` block, I'd say just remove the `beforeEach` and put the line

```ts
cy.getByTestID('homepage-wizard-language-tile--arduino').click()
```
at the top of the `it('runs through the arduino wizard pages'` block

Reply:
I'll check that it's not required for the other describe block (`describe(Subway Nav Bar)`, if not I can move it 👍

Reply:
I checked and we need that before each for the Subway Nav Bar test block.